emready/protocols/protocol_emready_sharpening.py

```diff
@@ -120,10 +120,10 @@ def createOutputStep(self):
 
     def _validate(self):
         errors = []
-        if (48 < self.stride or self.stride < 10):
-            errors.append('`stride` should be within [10, 48].')
+        if (12 < self.stride < 48):
```

Comment:
It seems that errors will be reported for stride between 12 and 48, which is just in contrast to its original purpose (to make sure 12 <= stride <= 48).